pkg/generator/rps.go

```diff
@@ -0,0 +1,142 @@
+package generator
+
+import (
+	"fmt"
+	"github.com/vhive-serverless/loader/pkg/common"
+	"github.com/vhive-serverless/loader/pkg/config"
+	"math"
+	"math/rand"
+)
+
+func generateFunctionByRPS(experimentDuration int, rpsTarget float64) (common.IATArray, []int) {
+	iat := 1000000.0 / float64(rpsTarget) // μs
+
+	var iatResult []float64
+	var countResult []int
+
+	duration := 0.0 // ms
+	totalExperimentDurationMs := float64(experimentDuration * 60_000_000.0)
+
+	currentMinute := 0
+	currentCount := 0
+
+	for duration < totalExperimentDurationMs {
+		iatResult = append(iatResult, iat)
+		duration += iat
```

Comment:
So, for RPS mode there is only equidistant IAT with minute granularity? Why do we need this mode if we can use output of tools/trace_synthesizer with any set of IAT types and granularity?

Reply:
Because it's not that trivial to create a trace that can trigger some cold/warm start ratios. Knative is not responsive enough, so a workaround must be used.

Reply:
But you are creating this trace inside the loader itself. Dumping the counted invocations for each minute and replaying it will give you the same result.

Didn't get the point about knative not being responsive.